src/catchup/ApplyBucketsWork.cpp

```diff
@@ -193,7 +197,8 @@ ApplyBucketsWork::advance(BucketApplicator& applicator)
             << "Bucket-apply: " << mAppliedEntries << " entries in "
             << formatSize(mAppliedSize) << "/" << formatSize(mTotalSize)
             << " in " << mAppliedBuckets << "/" << mTotalBuckets << " files ("
-            << (100 * mAppliedSize / mTotalSize) << "%)";
+            << (100 * mAppliedSize / mTotalSize) << "%, "
```

Comment:
continuing conversation started in #1902 
> Continued in the bulk-operations PR #1906 . @MonsieurNicolas I felt it was helpful to get performance characteristics and since it's surfaced to field users as an INFO log message during catchup it might be an easily-compared signal about the difference in performance between our dev setups and the sort of IO subsystems our production users have. But if you feel strongly opposed I can drop it from that PR.

What I don't really like with this one is that it adds a new counter that cannot be reasoned about between runs (as it depends on which buckets got applied). We've been cleaning up metrics to allow people to import in bulk all metrics into systems like prometheus and this one fails the "usefulness" smell in a production environment.

If we want to give easy to read performance numbers in the logs, I would make the following changes:
* make the numbers easier to compare by reporting on a per bucket basis. Basically reset metrics when we start a new bucket, that way we can compare bucket 5.snap from one run with the same bucket 5.snap from a different run. 
* report only at the end of a bucket (INFO), optionally report every "advance" (in DEBUG)
* don't use the global medida registry (as to not export those metrics) or simple don't use medida metrics (ie: record the time)
* potentially break things down by bucket type (live vs delete) - I suspect deletes are adding a lot of noise

Reply:
done